### ISSUE TITLE
Parser issue with empty comment

### ISSUE THREAD

**Issue by soliad** [bytedeco/javacpp#14]
There is an issue with the CommentBefore method:
```
Caused by: java.lang.StringIndexOutOfBoundsException: String index out of range: 3
        at java.lang.String.charAt(Unknown Source)
        at org.bytedeco.javacpp.tools.Parser.commentBefore(Parser.java:900)
        at org.bytedeco.javacpp.tools.Parser.enumeration(Parser.java:1974)
        at org.bytedeco.javacpp.tools.Parser.declarations(Parser.java:2191)
        at org.bytedeco.javacpp.tools.Parser.parse(Parser.java:2266)
        at org.bytedeco.javacpp.tools.Parser.parse(Parser.java:2361)
        at org.bytedeco.javacpp.tools.Builder.parse(Builder.java:64)
        at org.bytedeco.javacpp.tools.Builder.build(Builder.java:608)
        at org.bytedeco.javacpp.tools.BuildMojo.execute(BuildMojo.java:267)
        ... 21 more
```
It happen when there is an empty comment like just ///

The line 900
```java
if (s.charAt(3) == '<') {
```
need to check the size first:
```java
if (error.length() > 3 && s.charAt(3) == '<') {
```
Thanks

**Comment by saudet** [comment]
Thanks for reporting!! Let me know if you encounter any other issues, thanks.

**Comment by saudet** [comment]
Fix included in version 1.0. Thanks again for the fix!
